apis/locationMasterDataService/lmds.yaml

```diff
@@ -0,0 +1,328 @@
+swagger: '2.0'
+info:
+  version: '1.0'
+  title: Location Master Data Service
+  termsOfService: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+  contact:
+    email: admin@open-sunbird.org
+  description: >-
+    The Location Master Data Service APIs are for performing bulk upload of
+    location master data into Diksha Sunbird Platform. The master data is
+    provided in CSV format.
+  license:
+    name: GNU Affero General Public License v3.0
+    url: 'https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE'
+host: staging.open-sunbird.org
+basePath: /api/shaalakosh/v1
+schemes:
+  - https
+paths:
+  /location/state/upload:
```

Comment:
@manzarul 
The URIs are not appropriate as it does not fall in line with Sunbird terms .

The URIs are more specific to DIKSHA use case, but on Sunbird we should be using generic naming conventions

@bvinayakumar